### DISCUSSION TITLE
Not able to view the default blogs

### DISCUSSION THREAD

**Discussion by gnataraj** [alshedivat/al-folio#1668] [Q&A]
Hello, I am new to Jekyll and al-folio.

I am following the steps mentioned in [deployment](https://github.com/alshedivat/al-folio#deployment) to set up the al-folio with my GitHub account. After updating the _config.yml and the automatic GitHub action trigger, looks like deployment was successful. However, when I access the 'blog' page, I am getting **"Hello World" <\h1>** from nowhere! 

Below is the screenshot.
![image](https://github.com/alshedivat/al-folio/assets/133146/89233589-dab1-4aec-97c9-14d510e32422)
Can someone please suggest me if I am missing something while setting up the things.

**Comment by george-gca** [comment]
Is this still an issue? If yes, please provide more information like what did you already change in your repo.

**Reply by gnataraj-mf** [reply]
No, It is no more a issue now.  I found out the cause of this issue. 
I had a repo by name `blog` with just a `Hello World` page.
So whenever I hit https://gnataraj.github.io/blog/ , it was redirected to `blog `repo and that page was rendering.
I deleted the `blog ` repo, and everything stated working as expected!
